### ISSUE TITLE
[Feature Request] Support Popper Recording Options

### ISSUE THREAD

**Issue by Ltek** [syd711/vpin-studio#317]
Likely due to the very old version of VLC Popper uses, it has some serious problems with loading screens that are larger than 1080p. It stutters and pixelates on most of them. 

The recommended "fix" for any problem media has always been to convert to the 'known good' format Popper recommends, which uses specific FFMPEG conversion, provided by popper's author via BAT file script (see below)

Would be nice to have the ability to for Studio to...

- On import into Studio, 'auto convert' all media that is larger than 1080p
- a button on the Popper screen that allows converting a single video (for those who dont need to convert all videos)

```
setlocal EnableDelayedExpansion

CD /d %~dp0
if not exist "ffmpeg.exe" (
echo ***** ffmpeg.exe not found should be with batch
pause
exit
)


for /R %%f in (*.mp4) do (

ffmpeg -y -i "%%~df%%~pf%%~nf%%~xf" -c:v libx264 -preset fast -crf 22 -profile:v baseline -pix_fmt yuv420p -vf "scale=1920x1080,fps=30" -b:a 128k "PUPMediaoutput.mp4"

IF %ERRORLEVEL% EQU 0 (
del "%%~df%%~pf%%~nf%%~xf"
copy /y "PUPMediaoutput.mp4" "%%~df%%~pf%%~nf.mp4"
)

)
del "PUPMediaoutput.mp4"
pause

```

**Comment by syd711** [comment]
Actually we should support all video conversion scripts then.
The folder C:\vPinball\PinUPSystem\Recordings has a bunch of pre-defined scripts that could be triggered.

The jobs UI of the Studio needs a revamp before this though.

**Comment by Ltek** [comment]
bump... could really use this :)

Example... DL'd a new one yesterday from the forums, 4k loading video 70mb!
I didnt rescale it all and just change bitrate from 18000 (yea, ridiculously too high!) to 10500 then it saved at 38mb and ran smooth after that.
prob could go to 7000-8000 and no one would know the diff.

having that in Studio would be really nice... see the res and bitrate, select "Change Bitrate" or "Change Resolution" (or both) and  click 'go' :)

**Comment by syd711** [comment]
Is blocked by https://github.com/syd711/vpin-studio/issues/459

**Comment by syd711** [comment]
Fixed with 3.6.0
